index.html

```diff
@@ -44,16 +44,142 @@ <h2 class="car">I'm a software developer</h2>
+                            <li><a href="#">Ruby on rails</a></li>
+                            <li><a href="#">CSS</a></li>
+                            <li><a href="#">JavaScript</a></li>
+                            <li><a href="#">HTML</a></li>
+                        </ul>
+                        <input type="button" class="button" value="See Project">
+                    </div>
+                </div>
+                <div class="project-box">
+                    <img class="img-project3" alt="project3-image" src="img/test.png" />
+                    <div class="project-details">
+                        <h3>Multi-Post Stories Gain+Glory</h3>
+                        <ul>
+                            <li><a href="#">Ruby on rails</a></li>
+                            <li><a href="#">CSS</a></li>
+                            <li><a href="#">JavaScript</a></li>
+                            <li><a href="#">HTML</a></li>
+                        </ul>
+                        <input type="button" class="button" value="See Project">
+                    </div>
+                </div>
+                <div class="project-box">
+                    <img class="img-project4" alt="project4-image" src="img/test.png" />
+                    <div class="project-details">
+                        <h3>Multi-Post Stories Gain+Glory</h3>
+                        <ul>
+                            <li><a href="#">Ruby on rails</a></li>
+                            <li><a href="#">CSS</a></li>
+                            <li><a href="#">JavaScript</a></li>
+                            <li><a href="#">HTML</a></li>
+                        </ul>
+                        <input type="button" class="button" value="See Project">
+                    </div>   
+                </div>
+                <div class="project-box">
+                    <img class="img-project5" alt="project5-image" src="img/test.png" />
+                    <div class="project-details">
+                        <h3>Multi-Post Stories Gain+Glory</h3>
+                        <ul>
+                            <li><a href="#">Ruby on rails</a></li>
+                            <li><a href="#">CSS</a></li>
+                            <li><a href="#">JavaScript</a></li>
+                            <li><a href="#">HTML</a></li>
+                        </ul>
+                        <input type="button" class="button" value="See Project">
+                    </div>
+                </div>
+                
+
+            </section>
+        </div>
+
+        <div class="about">
+            <h2 class="name">About me</h2>
+            <h3> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</h3>
+
+            <input type="button" class="button" value="Get My Resume">
+
+            <section class="lang-container">
```

Comment:
- [ ] Please use the `<section>` element to wrap the about section as this is best practice for HTML semantics.

- [ ] Kindly add the background images as it is in the template so that your work can match the design template.
## Your headline
![headline](https://user-images.githubusercontent.com/94551484/185504543-57902ad0-0132-485a-8a8d-ec4293a399eb.png)
## design template
![req](https://user-images.githubusercontent.com/94551484/185504554-efa4fe30-1668-498e-ad1d-c54011de4ece.png)
## Your about section
![abt](https://user-images.githubusercontent.com/94551484/185504579-5d9890f6-74c8-42cb-9ffd-895d71d54b47.png)
## Design template
![reqabt](https://user-images.githubusercontent.com/94551484/185504593-feb2cad7-e582-414b-a6e1-fd5dcec8de5f.png)

- [ ] Kindly change the paragraph below the "About me" heading to a `<p>` tag element.